src/subscript/fmuobs/fmuobs.py

```diff
@@ -312,9 +312,14 @@ def fmuobs(
         if __MAGIC_STDOUT__ in (csv, yml, ertobs):
             raise SystemExit("Don't use verbose mode when writing to stdout")
         logger.setLevel(logging.INFO)
-
+        getLogger("subscript.fmuobs.parsers").setLevel(logging.INFO)
+        getLogger("subscript.fmuobs.writers").setLevel(logging.INFO)
+        getLogger("subscript.fmuobs.util").setLevel(logging.INFO)
     if debug:
         logger.setLevel(logging.DEBUG)
+        getLogger("subscript.fmuobs.parsers").setLevel(logging.DEBUG)
+        getLogger("subscript.fmuobs.writers").setLevel(logging.DEBUG)
+        getLogger("subscript.fmuobs.util").setLevel(logging.DEBUG)
 
```

Comment:
I think you can make this shorter by avoiding repetition:

```
loglevel = logging.INFO
if debug:
     loglevel = logging.DEBUG

logger.setLevel(loglevel)
getLogger("subscript.fmuobs.parsers").setLevel(loglevel)
...
```

Reply:
fixed